site/content/blog/2022-07-01-whats-new-in-svelte-july-2022.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "What's new in Svelte: July 2022"
+description: "Faster SSR, Language tools improvements and something special..."
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With [something special](https://twitter.com/SvelteSociety/status/1539284972544180230?s=20&t=3Qoi371NrL9LP6xMOF63ag) dropping today along with faster SSR and a number of new language tools features, there's a lot to cover in this month's newsletter...
+
+So let's dive in!
+
+## What's new in Svelte & Language Tools
```

Comment:
I thought it might be useful to share an update about OpenCollective funding

```suggestion
## OpenCollective funding drives Svelte forward

Svelte supporters have donated approximately $80,000 to [the project on OpenCollective](https://opencollective.com/svelte). We're happy to share that the funds are being drawn on to move Svelte forward in a meaningful way. [@gtm-nayan](https://github.com/gtm-nayan) has begun triaging and fixing SvelteKit issues this past month as a paid contributor to the project to help us get SvelteKit to a 1.0 level of stability! @gtm-nayan has been an active member of the Svelte community for quite some time and is well known for writing the bot that helps keep our Discord server running. We're happy that this funding has allowed Svelte to get much more of his time.

We will also be utilizing OpenCollective funds to allow Svelte core maintainers to attend Svelte Summit in person this fall.

## What's new in Svelte & Language Tools
```